src/hotspot/share/gc/g1/g1FullGCMarker.cpp

```diff
@@ -25,14 +25,16 @@
 #include "precompiled.hpp"
 #include "classfile/classLoaderData.hpp"
 #include "gc/g1/g1FullGCMarker.inline.hpp"
+#include "gc/g1/g1RegionMarkStatsCache.inline.hpp"
```

Comment:
Shouldn't need the `inline.hpp` here, `.hpp` should be enough.

Reply:
Thanks Stefan.